spec/book_spec.rb

```diff
@@ -0,0 +1,81 @@
+require 'rails_helper'
+
+describe Book, type: :model do
+  #  before(:all) do
+  #    @book = create(:book)
+  #  end
+
+  it do
+    should validate_presence_of(:title)
+  end
+  it do
+    should validate_presence_of(:author)
+  end
+
+  #  subject(:book) do
+  #    create(:book)
+  #  end
+
+  #  let(:title)                { Faker::Book.title }
+  #  let(:author)               { Faker::Book.author }
+  #  let(:publisher)            { Faker::Book.publisher }
+  #  let(:genre)                { Faker::Book.genre }
+  #  let(:image)                { ':O' }
+  #  let(:year)                 { '2018' }
+
+  describe '#create' do
+    context 'When the book is properly set' do
+      let(:book) { create(:book) }
+      it do
+        expect(book).to be_valid
+      end
+    end
+
+    context 'When the author is nil' do
```

Comment:
Maybe you can though about somthing DRYier to execute all of this test which seems very similars.